contrib/debian/manpages/bitcoin-cli.1

```diff
@@ -0,0 +1,52 @@
+.TH BITCOIN-CLI "1" "February 2015" "bitcoin-cli 0.10" 
+.SH NAME
+bitcoin-cli \- the remote procedure call client for bitcoin core. 
+.SH SYNOPSIS
+bitcoin-cli [options] <command> [params] \- Send command to Bitcoin Core. 
+.TP
+bitcoin-cli [options] help \- List Bitcoin Core commands.
+.TP
+bitcoin [options] help <command> \- Get help for a command.
+.SH DESCRIPTION
+This manual page documents the bitcoin-cli program. bitcoin-cli is the RPC client used to send commands to Bitcoin Core.
+
+.SH OPTIONS
+.TP
+\fB\-conf=\fR<file>
+Specify configuration file (default: bitcoin.conf)
+.TP
+\fB\-datadir=\fR<dir>
+Specify data directory
+.TP
+\fB\-testnet\fR
+Use the test network
+.TP
+\fB\-regtest\fR
+Enter regression test mode, which uses a special chain in which blocks can be solved instantly. This is intended for regression testing tools and app development.
+.TP
+\fB\-rpcuser=\fR<user>
+Username for JSON\-RPC connections
+.TP
+\fB\-rpcpassword=\fR<pw>
+Password for JSON\-RPC connections
+.TP
+\fB\-rpcport=\fR<port>
+Listen for JSON\-RPC connections on <port> (default: 8332 or testnet: 18332)
+.TP
+\fB\-rpcconnect=\fR<ip>
+Send commands to node running on <ip> (default: 127.0.0.1)
+.PP
+SSL options: (see the Bitcoin Wiki for SSL setup instructions)
+.TP
+\fB\-rpcssl\fR=\fI1\fR
+Use OpenSSL (https) for JSON\-RPC connections
+.TP
+\-?
+This help message
+
+.SH "SEE ALSO"
+bitcoind.d(1), bitcoin.conf(5)
+.SH AUTHOR
+This manual page was written by Ciemon Dunville. Permission is granted to copy, distribute and/or modify this document under the terms of the GNU General Public License, Version 3 or any later version published by the Free Software Foundation.
```

Comment:
I could, but I've used the same license as the other Debian man pages, and
as I'm sure you know master/contrib/debian/copyright is fairly big already;
arguably it doesn't need me dropping more lines into it?

On 20 February 2015 at 17:57, Wladimir J. van der Laan <
notifications@github.com> wrote:

> In contrib/debian/manpages/bitcoin-cli.1
> <https://github.com/bitcoin/bitcoin/pull/5809#discussion_r25088467>:
>
> > +.TP
> > +\fB\-rpcconnect=\fR<ip>
> > +Send commands to node running on <ip> (default: 127.0.0.1)
> > +.PP
> > +SSL options: (see the Bitcoin Wiki for SSL setup instructions)
> > +.TP
> > +\fB\-rpcssl\fR=\fI1\fR
> > +Use OpenSSL (https) for JSON\-RPC connections
> > +.TP
> > +\-?
> > +This help message
> > +
> > +.SH "SEE ALSO"
> > +bitcoind.d(1), bitcoin.conf(5)
> > +.SH AUTHOR
> > +This manual page was written by Ciemon Dunville. Permission is granted to copy, distribute and/or modify this document under the terms of the GNU General Public License, Version 3 or any later version published by the Free Software Foundation.
>
> Thanks for adding a manpage for bitcoin-cli! However, can you please
> license it under the same license as the rest of the repository? (which is
> the MIT license, see COPYING)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/5809/files#r25088467>.
>